src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -105,7 +105,12 @@ public class GenevaExporterOptions
                     case string vs:
                         break;
                     default:
-                        throw new ArgumentException($"Type: {entry.Value.GetType()} is not supported. Only bool, byte, sbyte, short, ushort, int, uint, long, ulong, float, double, and string are the supported types for PrepopulatedFields values.");
+                        if (entry.Value == null)
+                        {
+                            throw new ArgumentException($"Null is not allowed (key = `{entry.Key}`).");
```

Comment:
Do we want to do something similar to what we have for `TableNameMappings` to stay consistent? https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs#L49

Reply:
Good point, I've made the updates.